perception/object_recognition/detection/roi_cluster_fusion/include/roi_cluster_fusion/roi_cluster_fusion_nodelet.hpp

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2020 Tier IV, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+#include <rclcpp/rclcpp.hpp>
+#include <tf2_ros/buffer.h>
+#include <tf2_ros/transform_listener.h>
+#include <memory>
+#include "autoware_perception_msgs/msg/dynamic_object_with_feature.hpp"
+#include "autoware_perception_msgs/msg/dynamic_object_with_feature_array.hpp"
+#include "message_filters/pass_through.h"
+#include "message_filters/subscriber.h"
+#include "message_filters/sync_policies/approximate_time.h"
+#include "message_filters/synchronizer.h"
+#include "sensor_msgs/msg/camera_info.hpp"
+
+namespace roi_cluster_fusion
+{
+class RoiClusterFusionNodelet : public rclcpp::Node
+{
+public:
+  RoiClusterFusionNodelet(const rclcpp::NodeOptions & options);
+
+private:
+  virtual void onInit();
```

Comment:
Please remove, this is causing the bug when running the node
```suggestion
```

Reply:
this is fixed